deploy/charts/secret-manager/templates/deployment.yaml

```diff
@@ -60,6 +63,13 @@ spec:
           {{- range $arg := .Values.extraArgs }}
           - {{ $arg }}
           {{- end }}
+          env:
+          {{- range $name, $value := .Values.env }}
```

Comment:
Rather than using custom helm syntax for environment variables, I would think it makes more sense just to embed the environment value within the container spec. This allows references from configMaps, secrets or anything else.



````yaml
env:
  FOO: "first-value"
  BAR: "second-value"
````

vs 
```yaml
env:
- name: "FOO"
  value:  "first-value"
- name: "BAR"
  valueFrom:
    configMapKeyRef:
      name: configmap
      key: selector
```